src/components/layout.tsx

```diff
@@ -29,7 +29,7 @@ const Layout: React.FC<Props> = ({
 }) => {
     return (
         <>
-            <Head title={title} description={description} meta={meta} />
+            <Head title={title ?? "Untitled Page"} description={description} meta={meta} />
```

Comment:
Do we want to display "Untitled page" as a fallback? Currently the behaviour is that if you pass `undefined` as title it will fall back to "Christ Church Mayfair", isn't that the behaviour we want?

Reply:
yup - me just meddling. Will adjust